autofit/graphical/messages/abstract.py

```diff
@@ -123,33 +135,37 @@ def sub_natural_parameters(self, other: "AbstractMessage"
     _multiply = sum_natural_parameters
     _divide = sub_natural_parameters
 
-    def __mul__(self, other: "AbstractMessage") -> "AbstractMessage":
-        if np.isscalar(other):
+    def __mul__(self, other: Union["AbstractMessage", int, float]) -> "AbstractMessage":
```

Comment:
There's actually Numbers abstract base type which you can use to express int or float

Reply:
Set it to numbers.Real